### ISSUE TITLE
Metro Sales Summary Report Extraction

### ISSUE THREAD

**Issue by Sakthi1502** [beezlabs-org/automation-dev#148]
Customer has requested for new reports for 9 Categories to be extracted from Metro Portal which will be an enhancement request.

**Comment by Sakthi1502** [comment]
Update: Have requested some clarifications from Lissa yesterday post which I can start working on PDD.

**Comment by Sakthi1502** [comment]
Update: Had a call with Lissa to close some clarifications. Lissa has another request for a proposal to implement data validation for the same process which needs to be aligned with tech team as well.

**Comment by Sakthi1502** [comment]
- Lissa is fine to start with automating the process as first step and keep the data validation part as a separate initiative. The complexity for automating the process would be simple (2 week effort). 
- Data validation part will be a larger scope since the expectation from client is to implement it across all retailers which will be aligned with internal teams based on the call I had with client today.

**Comment by Dean13b** [comment]
Client expectation is to close it asap

**Comment by Sakthi1502** [comment]
Dates for PDD Completion and development to be given once we receive the budget approval

**Comment by Dean13b** [comment]
Naveen to take this delivery

**Comment by Dean13b** [comment]
file renaming and csv conversion is pending. to be completed by EOD today

**Comment by Sakthi1502** [comment]
Developer is facing issue during Metro portal login. The login creds were working fine during daytime but he is facing issue while testing now. This has been raised as an email to Lissa. Developer Testing is on hold due to this. cc: @dominicceo @mbharathm @Dean13b 

![Image](https://github.com/beezlabs-org/automation-dev/assets/65459771/3468896d-31bd-41a3-a19c-a1728cc4b7c3)

**Comment by Sakthi1502** [comment]
Update: Code Review to be done. @AmitRanjanBasu to confirm once code review is done. SIT has been started in parallel.

**Comment by Sakthi1502** [comment]
Update: Have identified couple of issues after performing SIT. @NaveenKumarD8 to fix and get back on the issues to proceed further.

- Naming convention truncating the year value in the report name
- Customer type not reflected in the final report data

**Comment by NaveenKumarD8** [comment]
The above Mentioned Issues were completed and Tested.

**Comment by koushik-ac** [comment]
Code review completed by @AmitRanjanBasu. @Sakthi1502 to confirm the status of the changes suggested by Amit.

**Comment by Sakthi1502** [comment]
Update: Test results along with data have been shared with client for review and UAT Signoff.

**Comment by Dean13b** [comment]
UAT call planned by 8th January

**Comment by Sakthi1502** [comment]
Update: Lissa has come up with a feedback that they are facing error while loading the data supplied. Will check on this and get back.

**Comment by Sakthi1502** [comment]
Update: Had a call with Lissa on Friday regarding the changes suggested. The changes were not part of initial requirement and Lissa said she needs to check with Cicille on the changes and get back. We have sent a follow up email on this today as well. Once we have a go-ahead from Lissa, we will implement the changes.

**Comment by Dean13b** [comment]
Expecting inputs from enduser by 2nd week of february
